### PR TITLE
Simplify sentieon test license server usage

### PR DESCRIPTION
**Pull request by maxulysse**
Replace https://github.com/nf-core/sarek/pull/1237 and https://github.com/nf-core/sarek/pull/1238

Idea is to move it to configs https://github.com/nf-core/configs/pull/551
To allow better control on how we can deal with this test license server for all nf-core pipelines / modules that needs it.
Plus being more future proof as we can modify configs without having to modify pipelines/modules

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- [#1242](https://github.com/nf-core/sarek/pull/1242) - Simplify sentieon nf-core test lisence usage
```
Comment (maxulysse) [inline CHANGELOG.md:16]:
```suggestion
- [#1242](https://github.com/nf-core/sarek/pull/1242) - Simplify sentieon nf-core test license usage
```

File: tests/test_umi.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@
     # text-based file changes md5sums on reruns
 
 - name: Run Sentieon-FGBio UMI combination test
-  command: nextflow run main.nf -profile test_cache,umi --outdir results --aligner "sentieon-bwamem"
+  command: nextflow run main.nf -profile test_cache,sentieon_test_licence,umi --outdir results --aligner "sentieon-bwamem"
```
Comment (maxulysse) [inline tests/test_umi.yml:55]:
```suggestion
  command: nextflow run main.nf -profile test_cache,sentieon_test_licence,umi --sentieon_extension --outdir results --aligner "sentieon-bwamem"
```

File: tests/test_sentieon_aligner_bwamem.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 - name: Run sentieon bwamem
-  command: nextflow run main.nf -profile test_cache --aligner sentieon-bwamem --save_reference --outdir results
+  command: nextflow run main.nf -profile test_cache,software_license --sentieon_extension --aligner sentieon-bwamem --save_reference --outdir results
```
[the remark this answers is not in the document]
Comment (maxulysse) [reply tests/test_sentieon_aligner_bwamem.yml:2]: yeah, a bit of a mess, but future proof whenever we add more tools

[the remark this answers is not in the document]
Comment (maxulysse) [reply tests/test_sentieon_aligner_bwamem.yml:2]: Anyway we will be refactoring all the tests soon enough, so we can rethink that bit

File: tests/test_umi.yml [hunk 4]
```diff
@@ -52,7 +52,7 @@
     # text-based file changes md5sums on reruns
 
 - name: Run Sentieon-FGBio UMI combination test
-  command: nextflow run main.nf -profile test_cache,umi --outdir results --aligner "sentieon-bwamem"
+  command: nextflow run main.nf -profile test_cache,sentieon_test_licence,umi --sentieon_extension --outdir results --aligner "sentieon-bwamem"
```
Comment (maxulysse) [inline tests/test_umi.yml:55]:
```suggestion
  command: nextflow run main.nf -profile test_cache,software_license,umi --sentieon_extension --outdir results --aligner "sentieon-bwamem"
```

**Review by asp8200 — APPROVED** [review]

**Review by FriederikeHanssen — COMMENTED** [review]
How come the new parameter is not part of the schema? When is it supposed to be used? For anyone, just for testing?

**Comment by maxulysse** [comment]
> How come the new parameter is not part of the schema? When is it supposed to be used? For anyone, just for testing?

because it's a params from configs: https://github.com/nf-core/configs/pull/551

**Pull request merged**